webapp/app/forms/fields.py

```diff
@@ -63,6 +72,37 @@ def _value(self):
         return self.data.split('-') if self.data else ''
 
 
+class SteuerlotseDateWidget(MultipleInputFieldWidget):
+    separator = ''
+    input_field_lengths = [2, 2, 4]
+    input_field_labels = [_('date-field.day'), _('date-field-month'), _('date-field-year')]
+
+
+class SteuerlotseDateField(DateField):
+
+    def __init__(self, **kwargs):
+        kwargs.setdefault('format', "%d %m %Y")
+
+        if kwargs.get('render_kw'):
+            if kwargs['render_kw'].get('class'):
+                kwargs['render_kw']['class'] = kwargs['render_kw']['class'] + " date_input form-control"
+            else:
+                kwargs['render_kw']['class'] = "date_input form-control"
+            if 'example_input' not in kwargs['render_kw']:
+                kwargs['render_kw']['example_input'] = _('fields.date_field.example_input.text')
```

Comment:
and also of this conditional
```suggestion
                kwargs['render_kw']['example_input'] = kwargs['render_kw'].get('example_input', _('fields.date_field.example_input.text'))
```

Reply:
Wow, I am impressed. 👍

Reply:
😊